src/System.Security.Cryptography.X509Certificates/pkg/ValidationSuppression.txt

```diff
@@ -0,0 +1 @@
+PermitHigherCompatibleImplementationVersion
```

Comment:
Consider adding a comment here.  We are intentionally leaving references at RTM version so that we don't version past 1.1.  In 1.1 buildtools this is default behavior and permitted by validation but in 1.0 we need to force it by not changing the ref assembly version and suppressing the validation mismatch.